<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table_factureros</class>
 <widget class="QWidget" name="table_factureros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listado Factureros</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_test">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Texto de Prueba&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="6">
       <widget class="QTableView" name="table"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="btn_edit">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="2">
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>Borrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
